src/DotNetStatus/Common/AnalyzerInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Fx.Portability
+{
+    public class AnalyzerInfo
```

Comment:
This name is not descriptive enough. It's really an object to hold info about the compatibility diagnostics, so maybe 'CompatibilityDiagnosticInfo'